src/Services/Importers/Chunk.php

```diff
@@ -59,6 +59,10 @@ public function __construct(
 
     public function run(): void
     {
+        if ($this->dataImport->fresh()->status === Statuses::Canceled) {
```

Comment:
let's do this one level higher, in the job. The job serializes models so there is not need for fresh.